include/ensmallen_bits/utility/randn.hpp

```diff
@@ -0,0 +1,72 @@
+/**
+ * @file randn.hpp
+ * @author Marcus Edel
+ *
+ * A simple `randn` wrapper that based on the data type forwards to
+ * `coot::randn` or `arma::randi`.
+ *
+ * ensmallen is free software; you may redistribute it and/or modify it under
+ * the terms of the 3-clause BSD license.  You should have received a copy of
+ * the 3-clause BSD license along with ensmallen.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef ENSMALLEN_UTILITY_RANDN_HPP
+#define ENSMALLEN_UTILITY_RANDN_HPP
+
+namespace ens {
+
+template<typename ElemType>
+typename std::enable_if<coot::is_coot_type<ElemType>::value, ElemType>::type
+randn(const size_t rows, const size_t cols)
```

Comment:
I think that the RNGs I just merged to bandicoot should match the Armadillo API, so I *think* you should be able to remove these functions and just call `randn<MatType>(...)` directly, etc.